.github/pull_request_template.md

```diff
@@ -9,8 +9,7 @@ Fixes #<issue number>
 - [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
 - [ ] Documentation added
 - [ ] Tests updated
-- [ ] `CHANGELOG.md` updated
-  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
```

Comment:
why don't we remove the action that was updating the release notes with this PR as well?

Reply:
done, I also added a note that we need to update the docs about updating the release notes as well, I'm not sure what changes we need to make there at the moment